Maui/MyMauiApp/MainPage.xaml.cs

```diff
@@ -12,10 +13,13 @@ public MainPage()
 	private void OnCounterClicked(object sender, EventArgs e)
 	{
 		count += 5;
+		Rotation += 90;
 
 		if (count <= 100)
-			CounterBtn.Text = $"Clicked {count} times";
-
+        {
```

Comment:
Take a look here https://www.mono-project.com/community/contributing/coding-guidelines/
specifically at the `if` statement part! ;)
@LeandraHall this is a good resource for you to look at as well!